server/routes/shared_api/choropleth.py

```diff
@@ -72,6 +72,8 @@
     "EurostatNUTS2": "geoJsonCoordinatesDP2",
     "EurostatNUTS3": "geoJsonCoordinatesDP1",
     "IPCCPlace_50": "geoJsonCoordinates",
+    "City": "geoJsonCoordinates",
```

Comment:
Are we sure we want this not DP1/DP2?

Reply:
Based on a quick BQ search, it looks like there are some cities that don't have a DP geojsons available. I can add logic to select D1/DP2 if available in a future PR?

Reply:
We might need to compute it then.